src/main/java/seedu/address/model/event/StartTime.java

```diff
@@ -15,10 +15,13 @@ public class StartTime {
 
     public static final String MESSAGE_CONSTRAINTS = "Start time must be in format: hh/mm/AM or hh/mm/PM";
 
-    //for checking if valid input date format and for changing to storage friendly format
-    private static final DateTimeFormatter checkAndLogFormatter = DateTimeFormatter
+    //for checking if valid input date format
+    private static final DateTimeFormatter checkFormatter = DateTimeFormatter
             .ofPattern("[hh/mm/a][h/mm/a][hh/m/a][h/m/a]");
```

Comment:
you may need to change this line for it to work on mac, 
```suggestion
            .ofPattern("[HH:MM][hh/mm/a][h/mm/a][hh/m/a][h/m/a]");
```
or 
```suggestion
            .ofPattern("[HH:MM]");
```
but this will change the time format to `HH:MM` completely, i.e. the time format can only be something like `14:00`, and some test cases need to be changed.